src/measure_theory/set_integral.lean

```diff
@@ -179,6 +179,10 @@ h.mono_measure $ measure.restrict_le_self
 lemma integrable.integrable_on' (h : integrable f (μ.restrict s)) : integrable_on f s μ :=
 h
 
+lemma integrable_on.restrict (h : integrable_on f s μ) (hs : measurable_set s) :
+  integrable_on f s (μ.restrict t) :=
+by rw [integrable_on, measure.restrict_restrict hs]; exact h.mono_set (inter_subset_left _ _)
```

Comment:
```suggestion
by { rw [integrable_on, measure.restrict_restrict hs], exact h.mono_set (inter_subset_left _ _) }
```
We avoid semicolons, unless there are multiple goals on which one wants to apply the same tactic.